lua/tardis/languages/en.lua

```diff
@@ -359,6 +361,15 @@ T.Phrases = {
     ["Settings.Sections.Global.EnableHealth.Description"] = "Should the TARDIS have health and take damage?",
     ["Settings.Sections.Global.MaxHealth"] = "Max Health",
     ["Settings.Sections.Global.MaxHealth.Description"] = "Maximum ammount of health the TARDIS has",
+    ["Settings.Sections.Global.WaitLongForRepair"] = "Enable long repair",
+    ["Settings.Sections.Global.WaitLongForRepair.Description"] = "Should the player have to wait 60 seconds for repair to finish?",
```

Comment:
This language is unclear as the repair takes 'up to' 60 seconds depending on the amount of health being repaired and max health. I think it would be better to instead change it to Quick Repair and invert the setting and simply describe it as quicker repair times

Reply:
This was fixed in e748a8cf338fed20c33127030d0791464cda6d5c